### PR TITLE
Initial addition of bed-reader.

### PR DESCRIPTION
**Pull request by jeromekelleher**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jeromekelleher** [comment]
OK, looks like this is working! @CarlKCarlK, would you mind taking a look please? There's a little bit of monkey-see-monkey do on my part as I've not used Rust like this before, so I based the recipe on the [pysynetc](https://github.com/conda-forge/pysyntect-feedstock). 

Also, can you confirm that you are happy to be named as a maintainer please?

**Comment by CarlKCarlK** [comment]
Great! I'll take a look later today.

Yes, I confirm that I'm happy to be the maintainer.


  *   Carl

**Comment by CarlKCarlK** [comment]
Jerome,

Everything looks sensible. Is there an easy way for me to test a conda-forge pull request? (Can I "conda install" from one?)

Thanks for your work on this.


  *   Carl


From: Carl KADIE
Sent: Wednesday, April 07, 2021 8:00 AM
To: conda-forge/staged-recipes ***@***.***>; conda-forge/staged-recipes ***@***.***>
Cc: Mention ***@***.***>
Subject: RE: [conda-forge/staged-recipes] Initial addition of bed-reader. (#14471)

Great! I'll take a look later today.

Yes, I confirm that I'm happy to be the maintainer.


  *   Carl

**Comment by jeromekelleher** [comment]
> Everything looks sensible. Is there an easy way for me to test a conda-forge pull request? (Can I "conda install" from one?)

I'm not sure if there's a way to try installing from the built packages before they get deployed. In my experience the CI is pretty strong, and if everything passes it's very unlikely that there's anything catastrophically wrong with the package. The simplest way forward is usually to get an initial package deployed (so that you can run ``conda install bed-reader``) and then make any tweaks with follow-up PRs to the official "feedstock".

The next step, if you agree, is to request one of the conda-forge administrators to merge this. Once the official feedstock repo exists then, we can update the package autonomously. Are you happy for this to go ahead? (It might take a few days for the admins to review and merge this PR, as there isn't that many of them and they are pretty stretched.)

**Comment by CarlKCarlK** [comment]
Yes, I agree.

**Comment by jeromekelleher** [comment]
@conda-forge/staged-recipes this is ready for final review and merge :heart:

File: recipes/bed-reader/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{% set name = "bed-reader" %}
+{% set version = "0.2.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # Can't use the standard pattern because of the hypen/underscore remapping.
+  url: https://files.pythonhosted.org/packages/source/b/bed-reader/bed_reader-{{ version }}.tar.gz
```
Comment (ocefpaf) [inline recipes/bed-reader/meta.yaml:10]: Can you do `https://pypi.io/packages/source/b/bed-reader/bed_reader-{{ version }}.tar.gz` instead? https://pypi.io/ will redirect to files.pythonhosted.org but the former seems to be more stable than the latter.

Comment (jeromekelleher) [reply recipes/bed-reader/meta.yaml]: Done! Thanks for taking a look @ocefpaf

**Pull request merged**